ci-operator/config/openshift/os/openshift-os-master.yaml

```diff
@@ -2,7 +2,7 @@ base_images:
   coreos_coreos-assembler_49:
```

Comment:
Should we go back to calling this `_latest`?

Reply:
Yep see the `latest` changes below